sites/docs/pages/components/big-value.md

```diff
@@ -1,180 +1,199 @@
----
-title: Big Value
-sidebar_position: 1
```

Comment:
deleting this throws off the sort in the sidebar.

was it intentional to delete all this content below also?

Reply:
GPT refactor run amok, change incoming for this one and a couple others.